lambda/getBasicInformationLambda.ts

```diff
@@ -74,29 +87,67 @@ export const handler = async (
 			],
 		})
 	}
+	const timeSpans = HistoricalDataTimeSpans[timeSpan]
+	if (timeSpans !== undefined) {
+		const result = await getBinInterval(ts, dbName, tableName)(
+			timeSpans.binIntervalMinutes,
+			timeSpans.durationHours,
+			iccid,
+		)
+		const measurements = []
+		for (const measurement of result) {
+			measurements.push({
+				timestamp: measurement.time,
+				usedBytes: measurement['measure_value::double'],
+			})
+		}
+		if (measurements.length < 12) {
+			await queueJob({
+				QueueUrl: validIssuers[issuer.issuerIdentifierNumber] as string,
+				sqs,
+			})({ payload: iccid, deduplicationId: iccid })
+		}
```

Comment:
Why do we queue this again? This could create an endless loop.

Reply:
The intention is to trigger a new fetch from the vendors API if we have less than 12 measurements (< 1 hour of measurements with 5min interval), but I see that that only applies for the 'lastHour' case and should probably be changed. 

If I remove it we are never getting any new data from the history request. If I can assume that the first req are getting requested every 5 minutes (APIURL/sim/<iccid>) this will be fetched in the background and I don't need to do anything. But if the history request is the only one getting requested I need to use the queue to trigger a new fetch for this case.

Reply:
Why are we not getting new data? The user will request the history again in 5 minutes when observing the device in the web app...

Reply:
If i remove these lines, the history request will only get the data that is available in Timestream and return those measurements. Then the job is done. By queueing we ensure that the data is also updated after the request. 

The first request on the other hand will use the queue to trigger a new fetch if the data is old. But if only the history request is used this will never happen.

I can check the timestamp from the results from the history request and queue if the data is older than 5min instead?

Reply:
This couples fetching of the data to the data returned by the API, instead it should be defined by whether a SIM was recently accessed.

If SIM was access withing the last hour: fetch history from vendor every 5 minutes, otherwise once a day (or whichever interval preserves the 5 minute granularity) for maximum of 30 days.

Reply:
Could the logic I have now work? I'm not sure if that was exactly what you were looking for, but the queue you commented on is removed and the updates are handled by the previous logic where you would get updates every 5 minutes by polling.